src/librustc_infer/infer/higher_ranked/mod.rs

```diff
@@ -81,7 +79,12 @@ impl<'a, 'tcx> InferCtxt<'a, 'tcx> {
     where
```

Comment:
@lcnr had some questions about this function. The doc comment appears to be outdated.